appengine/Rakefile

```diff
@@ -0,0 +1,19 @@
+$VERBOSE = true
+
+task :default => [:test]
+
+task :build do
+  ::Dir.chdir(::File.dirname(__FILE__)) do
+    puts("docker build -t appengine-ruby-base .")
```

Comment:
Verified and changed to `sh` in https://github.com/GoogleCloudPlatform/ruby-docker/pull/24